pkg/providers/amifamily/ami.go

```diff
@@ -79,25 +79,25 @@ func (a AMIs) Sort() {
 	})
 }
 
```

Comment:
nit: Should the amiProvider method be called `Get()` or `List()`? I'd recommend `List()` to align the call with the other providers (subnets and security groups)

Reply:
Also, do we need to `Sort()` in the `Get()` call anymore? If we are sorted outside the function and we assume that the status is ordered by creationDate already, do we have a need to do a second sort? Seems like we could just rely on the sort in the nodeClass controller.

Reply:
The status controller does not currently sort the AMIs by creation date

Reply:
But it could, right? Should it? Seems like it would be nice if that was all handled outside the List() call so we don't have an assumed dependency on the data that we return here.